Update api-management-using-with-vnet.md

jboeshart opened the pull request:
Added a note about testing from the portal and how the VNet DNS configuration needs to be able to properly resolve.

azurecla commented:
Hi __@begoldsm__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (begoldsm). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

nitinme commented:
#sign-off

cjgronlund commented:
@begoldsm @nitinme 
In the articles where you've added the "how to use" headings, I recommend deleting that heading and making the H3s below it H2s. Otherwise, you bury important steps in your heading hierarchy.

cjgronlund commented:
@begoldsm @nitinme 
I can accept for tomorrow's publish, but I recommend the change above at your earliest convenience. Preference? Fix now or fix later? Thanks!

begoldsm commented:
@cjgronlund I would say we can punt the recommended fixes for this PR. @nitinme can you take an action to address the recommended documentation changes in the next update?

cjgronlund commented:
@begoldsm @nitinme 
Thanks! Will do. This will be publishing in tomorrow morning's run. :)

jboeshart committed:
Update api-management-using-with-vnet.md

Added a note about testing from the portal and how the VNet DNS configuration needs to be able to properly resolve.

ktoliver committed:
capitalization for references to the Azure portal
